### ISSUE TITLE
Integrate joinmarket-webui

### ISSUE THREAD

**Issue by secondl1ght** [mynodebtc/mynode#628]
The joininbox integration for myNode has been great! There is now a new joinmarket web interface in development here: https://github.com/joinmarket-webui

The project is still in heavy development but is almost ready for it's first release. It would be great to have this new web interface added to myNode to compliment the CLI version of joinmarket that is already available.

Please let me know your thoughts on this, thanks.

**Comment by tehelsper** [comment]
Yeah, I'll definitely take a look once it gets a stable release out. Not sure if they are interested, but it may also get packaged in JoininBox, which is how I've been pulling in new JoinMarket releases lately.

**Comment by secondl1ght** [comment]
Sounds great, I am contributing to the project and I know the team is interested in having it on as many plug and play nodes as possible. I suggested myNode and everyone agreed that it would be good. This new web UI for joinmarket will be a massive UI/UX improvement for non-technical users. It will let users easily coinjoin but also become market makers to earn yield on their bitcoin, which I think many node users may be interested in!

We are hoping to have the first beta release out in the next couple weeks, it won't have all the planned functionality initially but we will work on adding in all the features. I'm sure there will be bug reports once we get users testing for real, so maybe the second beta release we could start looking at integrating with myNode after the initial issues are worked out. Although it seems to work quite well and stable in testing already.

**Comment by statusquont** [comment]
Came here to see if this was already requested, and glad to see it has been and is in consideration for future integrations!

**Comment by roarkjanis-resale** [comment]
I'm really happy to see this got added to MyNode but having some problems launching it as described in

https://github.com/joinmarket-webui/jam/issues/80
